interface/python_binding/README.md

```diff
@@ -24,4 +24,12 @@ cd ${PROJECT_DIR}/interface/python_binding
 KTX_RUN_TESTS=ON python3 buildscript.py
 ```
 
-If you are on a POSIX system (macOS or Linux), make sure libktx is on your `DYLD_LIBRARY_PATH` and `LD_LIBRARY_PATH`.
\ No newline at end of file
+If you are on a POSIX system (macOS or Linux), make sure libktx is on your `DYLD_LIBRARY_PATH` and `LD_LIBRARY_PATH`.
+
+> When building on macOS using against a universal CPython binary, such as that installed with the Xcode command-line tools (/usr/bin/python3), ld will issue a warning
```

Comment:
s/against //

Reply:
```suggestion
> When building on macOS against a universal CPython binary, such as that installed with the Xcode command-line tools (/usr/bin/python3), ld will issue a warning
```

I think "against" makes more sense, since the ABI has to match the Python interpreter being used.